helpers/account_helper.py

```diff
@@ -1,23 +1,51 @@
 import time
 from json import loads
+from typing import Any, Callable, TypeVar
 
-from retrying import retry
+from requests.models import Response
 
 from dm_api_account.apis.account_api import UserCredentials
 from dm_api_account.apis.login_api import UserLoginData
 from services.api_dm_account import ApiDmAccount
 from services.api_mailhog import ApiMailhog
 
+T = TypeVar('T')
 
-def retry_if_result_none(result):
-    """Return True if we should retry (in this case when result is None), False otherwise"""
+
+def retry_if_result_none(result: T) -> bool:
+    """
+    Проверяет, является ли результат None для механизма повторных попыток.
+    
+    Args:
+        result (T): Результат функции для проверки
+        
+    Returns:
+        bool: True если результат None, False в противном случае
+    """
     return result is None
 
 
-def retrier(function):
-    def wrapper(*args, **kwargs):
+def retrier(function: Callable[..., T]) -> Callable[..., T]:
```

Comment:
В питонах принято если может вернуться ошибка проставлять аннотацию, NoReturn
То есть типо того
```suggestion
def retrier(function: Callable[..., str | NoReturn]) -> Callable[..., str | NoReturn]:
```